bin/build.sh

```diff
@@ -47,11 +63,16 @@ mkdir build
 # Copy generated content into build folder
 cp -r documents build/
 cp -r edudoc build/
+# Copy all contents of the static folder into the build/documents folder
+cp -r static/*/ build/documents/
```

Comment:
Good news: this is unnecessary :)
See my comment [here](https://github.com/thewca/worldcubeassociation.org/pull/6259#issuecomment-864375467).